Lab0/README.md

```diff
@@ -50,7 +50,7 @@ You just installed your first extension.  Now, make sure to install at least the
 
 ![installdockerextension][installdockerextension]
 
-Once all the extensions are installed, click the *Refresh* button to restart VS Code.
+Once all the extensions are installed, click the *Reload* button to restart VS Code.
```

Comment:
To be validated for the specific extensions used here, but this step should not be needed anymore as extensions are installed without the need to "reload" since "patch I can't remember when"